client/src/components/Dashboard/League/Stepper.jsx

```diff
@@ -40,6 +42,48 @@ const Stepper = () => {
             alignItems="center"
             justifyContent="center"
         >
+            {isDesktop ? (
```

Comment:
[Optional] Nice job on the back arrow! This is completely optional, but you could probs just use a single component here and just use `size={isDesktop ? 'md' : 'xs'} boxSize={isDesktop ? 6 : null}` if you want to, but thats completely up to you.